types/mdx/mdx-tests.tsx

```diff
@@ -166,8 +169,7 @@ const MyComponentAliasAlias: typeof MyMDXComponent = MyComponentAlias;
             return <div {...props} />;
         },
         img: CustomImageComponent,
-        // $ExpectError
-        video: CustomImageComponent,
```

Comment:
Actually in retrospect I believe this assertion should be there.

It should be invalid to assign component which accepts `img` props to a `video` component.

Perhaps a different type of component can be defined to test `null` is allowed as a return value. I.e. a `CustomAnchorComponent`

Reply:
Ah right, you mean because it’s tested for every file extension? Maybe that’s a bit overkill, but it’s consistent to keep all those test cases in sync.

I do think it has added value to define a new component type for this specific test scenario, and also that it needs to be duplicated in the other test cases.